test/api/events-creation.test.js

```diff
@@ -103,6 +103,53 @@ describe('Events creation', () => {
         expect(res.body.data).toHaveProperty('questions');
         expect(res.body.data).toHaveProperty('organizers');
         expect(res.body.data).toHaveProperty('method');
+        expect(res.body.data).toHaveProperty('has_applications');
```

Comment:
Add a check that this is equal to true

Reply:
See suggestion here; https://github.com/AEGEE/events/pull/1246/files#r2265258488